### DISCUSSION TITLE
change dlna port

### DISCUSSION THREAD

**Discussion by Conkuist** [stashapp/stash#4759] [Q&A] [answered]
I ran into an issue where I can't run the stash dlna server and the xbvr dlna server at the same time.
I found out that both seem to use the port 1338 which seems to be the reason for my issue.
stash and xbvr seem to use the same dlna server which seems to be anacrolix/dms.
dms offers the abbility to change the http port and i was wondering how i would do that with stash running inside a docker container.

**Comment by DogmaDragon** [comment, marked as answer]
I believe the port is hardcoded. <https://github.com/stashapp/stash/blob/develop/internal/dlna/service.go#L141>
